cpp/IceGrid/simple/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(../../cmake/common.cmake)
 
 add_executable(client Client.cpp Hello.ice)
 slice2cpp_generate(client)
-target_link_libraries(client Ice::Ice Ice::IceGrid)
+target_link_libraries(client Ice::Ice Ice::IceGrid Ice::Glacier2)
```

Comment:
How does this work? do we still rely on `#pragma comment`?

Reply:
No, we don't rely on pragma comment. It doesn't cause any issues so we don't worry about it.